runtime/parachains/src/paras_inherent/mod.rs

```diff
@@ -260,22 +261,51 @@ impl<T: Config> Pallet<T> {
 			parent_header,
 			mut disputes,
 		} = data;
+		sp_io::init_tracing();
```

Comment:
Does this need to be guarded?

Reply:
No, we need to call it once before all else, IIUC.

Reply:
I mean guaraded by a feature flag?

Reply:
It is a no-op in the native runtime case or if `with-tracing` feature is not enabled in substrate.

Reply:
I'd prefer to guard this here, so we are not surprised in case something in substrate changes.